redbot/core/bot.py

```diff
@@ -690,6 +690,10 @@ def remove_cog(self, cogname: str):
         This should realistically only be used for responding using user provided
         input. (unfortunately, including usernames)
         Manually crafted messages which dont take any user input have no need of this
+        
+        Returns
+        -------
+        discord.Message
```

Comment:
I think this should also have a description as without it sphinx will still show `Returns:` field anyway but it will be empty:
![image](https://user-images.githubusercontent.com/6032823/66708069-435cdc00-ed4b-11e9-8047-2bc0444df51e.png)

```suggestion
        discord.Message
            The message that was sent.
```